### DISCUSSION TITLE
Tags Listing

### DISCUSSION THREAD

**Discussion by eswiig** [gotson/komga#1530] [Ideas]
### Describe your suggested feature

I know we can search tags and I just found out you have a list of tags as a loooong dropdown.

### Other details

Front end: 
It would be nice with a separate page like a tag index that lists all the tags A-Z, maybe 1-3 columns with a row on top of each letter of the alphabet or something/anything for a better desktop experience, instead of a long dropdown that you have to scroll.

Backend/Admin: 
It would also be nice to have a way to delete tags from a dedicated page instead of having to go into each title and remove the tag. Can be very tedious to do if you have a lot of tagged titles.

maybe another tab called tags here:
![image](https://user-images.githubusercontent.com/58608430/236701541-5f3aa916-715b-4b78-92d4-93abd1d3e9a7.png)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by Blazeflack** [comment]
From #1995 I would like to also include the wish for an Author's page.
